packages/nimble-components/src/rich-text/models/markdown-parser.ts

```diff
@@ -47,10 +52,44 @@ export class RichTextMarkdownParser {
             'escape'
         ]);
 
+        supportedTokenizerRules.validateLink = href => /^https?:\/\//i.test(href);
+
+        // In order to display encoded characters, non-ASCII characters, emojis, and other special characters in their original form,
+        // we bypass the default normalization of link text in markdown-it. This is done because we support only "AutoLink" feature in CommonMark flavor.
+        // "normalizeLinkText" method reference in markdown-it: https://github.com/markdown-it/markdown-it/blob/2b6cac25823af011ff3bc7628bc9b06e483c5a08/lib/index.js#L67C1-L86C2
+        supportedTokenizerRules.normalizeLinkText = url => url;
+
         return new MarkdownParser(
-            schema,
+            this.updatedSchema,
             supportedTokenizerRules,
             defaultMarkdownParser.tokens
         );
     }
+
+    private static getSchemaWithLinkConfiguration(): Schema {
+        return new Schema({
+            nodes: schema.spec.nodes,
+            marks: {
+                link: {
+                    attrs: {
+                        href: {},
+                        rel: { default: 'noopener noreferrer' }
+                    },
+                    inclusive: false,
```

Comment:
Same as above:
Will this inclusive configuration also need to change when hyperlink support is added?
If so can you add that in a comment?
If not can you describe why we are making this configuration change from the default?

Reply:
As mentioned in the [other comment](https://github.com/ni/nimble/pull/1496#discussion_r1325915858), we will take the right decision when link support is added. I have updated the issue checklist too.